shtc1/shtc1.c

```diff
@@ -203,6 +203,34 @@ void shtc1_enable_low_power_mode(uint8_t enable_low_power_mode) {
         enable_low_power_mode ? SHTC1_CMD_MEASURE_LPM : SHTC1_CMD_MEASURE_HPM;
 }
 
+int16_t shtc1_read_serial(uint32_t *serial) {
+    int16_t ret;
+    const uint16_t tx_words[] = {0x007B};
+    uint16_t *serial_words = (uint16_t *)serial;
```

Comment:
I'm not sure if we hurt strict-aliasing rules here. Could we add `-Wstrict-aliasing=1` to the Makefile? Also maybe just use a temporary uint16_t serial_words[2] array.

Reply:
yup